### PR TITLE
Include github settings for release upload to nexus 

### PR DESCRIPTION
**Pull request by Steffengreiner**
Include missing github.settings.xml necessary for the nexus release github action

### CODE REVIEW

**Comment by KochTobi** [comment]
LGTM

**Review by KochTobi — CHANGES_REQUESTED** [review]
Please remove Travis as well

**Review by KochTobi — APPROVED** [review]

**Comment by KochTobi** [comment]
> Please remove Travis as well

Nvm that, it was removed but not from the settings as a required check on the master branch.

File: .github.settings.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+            <username>${env.MAVEN_REPO_USERNAME}</username>
+            <password>${env.MAVEN_REPO_PASSWORD}</password>
+        </server>
+        <server>
+            <id>nexus-releases</id>
+            <username>${env.MAVEN_REPO_USERNAME}</username>
+            <password>${env.MAVEN_REPO_PASSWORD}</password>
+        </server>
+    </servers>
+</settings>
```
Comment (sven1103) [inline .github.settings.xml:14]: EOL!!!

**Pull request merged**